src/content/templates/ssp/xml/FedRAMP-SSP-OSCAL-Template.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="https://raw.githubusercontent.com/usnistgov/OSCAL/release-1.0/xml/schema/oscal_complete_schema.xsd" schematypens="http://www.w3.org/2001/XMLSchema" title="OSCAL complete schema"?>
+<?UseDebugMode value="false()"?>
```

Comment:
If this is placed (kinda permanently) into `FedRAMP-SSP-OSCAL-Template.xml`, that seems OK (reduces the `sch:report` noise. If that is your intent, should there be an adjacent comment that says something like `<-- alter the following to enable debug mode and thus enable multiple sch:report elements -->`. Without comment or documentation elsewhere, close scrutiny of the `ssp.sch` code would be necessary to determine the effect of the PI.

We should probably have a separate issue to update the documentation for use of remote resource usage and debug modes. No change is necessary here.